admin_ui/backend/api/live_status.py

```
@@ -0,0 +1,373 @@
+    lines.append("")
+    return "\n".join(lines) + "\n"
+
+
+@router.get("/live-status")
+async def get_live_status():
+    snapshot = await status_hub.snapshot()
+    if snapshot.get("components"):
+        return snapshot
+    try:
+        return await asyncio.wait_for(refresh_live_status(), timeout=_initial_probe_timeout_seconds())
+    except (asyncio.TimeoutError, Exception) as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)